### DISCUSSION TITLE
@load doesn't seem to work

### DISCUSSION THREAD

**Discussion by florian-sun** [alpinejs/alpine#4421] [1. Help] [answered]
This is an example. I got no alert when I load the page.

```
<body>   
  <div x-data="{count: 0}">
    <p @load="alert('hi')"></p>
  </div>
</body>
```

**Comment by SimoTod** [comment, marked as answer]
The load event is dispatched on the window object, not on the single html tags: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event

You can use the window modifier, see https://alpinejs.dev/directives/on#window

**Reply by ekwoka** [reply]
this isn't totally valid.

Object replacement elements (img, video, object) emit their own load events. Similarly, link and script elements

But yes, normal elements that don't load anything do not.

**Reply by SimoTod** [reply]
The example posted above was a p tag

**Reply by ekwoka** [reply]
Yes I see that. Just clarifying as many people not familiar with these events could misinterpret what you said.

**Comment by florian-sun** [comment]
OK, thanks
